### ISSUE TITLE
Retrieving unspliced sequence

### ISSUE THREAD

**Issue by austinv11** [openvax/pyensembl#313]
Hello, I may have missed it but is there a way to access a transcript's unspliced sequence and/or introns? 

I can infer intron positions based on the annotated exons, but I don't know how to then retrieve genomic sequences from these. Please let me know if you have any suggestions!

Thanks

**Comment by iskandr** [comment]
Closing as duplicate of #337 — once optional chromosome FASTA support lands, this becomes a one-liner against `genome.chromosome_sequences`. Subscribe to #337 for progress.
